sopel/test_tools.py

```diff
@@ -1,12 +1,11 @@
 # coding=utf-8
-"""This module has classes and functions that can help in writing tests.
+"""This module provided tools that helped to write tests.
 
-.. note::
+.. deprecated:: 7.1
 
-   This module formerly contained mock classes for bot, bot wrapper, and config
-   objects. Those are deprecated, and will be removed in Sopel 8.0. New code
-   should use the new :mod:`.mocks`, :mod:`.factories`, and
-   :mod:`.pytest_plugin` added in Sopel 7.0.
+    This module is replaced by the
+    :mod:`pytest plugin <sopel.tests.pytest_plugin>` for Sopel; it will be
+    **removed in Sopel 8**.
```

Comment:
This note doesn't fully cover the old mock classes, which are still here and also will be removed when this module gets deleted. Should mention the `mocks` & `factories` modules too.

Reply:
Yeah OK I'll update that.